packages/neos-ui-redux-store/src/UI/Remote/index.ts

```diff
@@ -59,6 +61,16 @@ const startDiscarding = () => createAction(actionTypes.START_DISCARDING);
  */
 const finishDiscarding = () => createAction(actionTypes.FINISH_DISCARDING);
 
+/**
```

Comment:
this seems unrelated? (everything in this file)

Reply:
Nah, it was a little improvement. Because when you click publishing really fast, it can happen that you save the link with the asset proxy and not the asset, as it was not imported yet. So I came up with this... so that the publishing is blocked until the import succeeds or fails.

Reply:
Also plan to add an API for that in future versions, as this is more a quick fix that misuse the saving state.
Would be nice to have an API to lock the publishing.